source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -121,14 +116,13 @@ def initialisation(self):
 
         for file_name in self.configuration_files:
             logging.getLogger(__name__).info("Initialisation, reading file: %s", file_name)
-            if file_name.endswith('.json'):
+            try:
                 with open(file_name) as f:
                     config_data = json.load(f)
                     self.register_instance(config_data)
-
-            else:
-                logging.getLogger(__name__).warn('impossible to init an instance with the configuration '
-                                                 'file %s', file_name)
+            except ValueError:
```

Comment:
hum I'm not sure about catching this, if the configuration file is not correct it might be better to fail fast